spec/models/recipe_spec.rb

```diff
@@ -2,14 +2,26 @@
 
 RSpec.describe Recipe, type: :model do
    
-  describe "validations" do
-    it {should validate_presence_of :name}
-    it {should validate_presence_of :complexity}
-    it {should validate_presence_of :genre}
-  end
-
-  describe "relationships" do
-    it {should have_many :recipe_ingredients}
-    it {should have_many(:ingredients).through(:recipe_ingredients)}
-  end
+    describe "validations" do
+        it {should validate_presence_of :name}
+        it {should validate_presence_of :complexity}
+        it {should validate_presence_of :genre}
+    end
+
+    describe "relationships" do
+        it {should have_many :recipe_ingredients}
+        it {should have_many(:ingredients).through(:recipe_ingredients)}
+    end
+  
+    describe "instance_methods" do
+        it '#total_cost' do
+            @recipe = Recipe.create!(name: "Ground Beef Stir Fry", complexity: 1, genre: "Asian")       
+            @recipe.ingredients.create!(name: "Ground Beef", cost: 6)
+            @recipe.ingredients.create!(name: "Onion", cost: 2)
+            @recipe.ingredients.create!(name: "Red Bell Pepper", cost: 1)
+
+            expect(@recipe.total_cost).to eq(9)
```

Comment:
Good model test here. You may want to consider testing in 3 cases overall: 

1. this method returns an expected value (what you have here; total_cost => 9)
2. This method can return a _different_ value than the first case (maybe another recipe that has a total cost of 5)
3. The _negative case_: this method can return `0` and not `[]` or `nil` or `false`, for example.